prov/efa/src/rdm/rxr_ep.c

```diff
@@ -1040,6 +1040,16 @@ static int rxr_ep_setopt(fid_t fid, int level, int optname,
 		if (ret)
 			return ret;
 		break;
+	case FI_OPT_EFA_SENDRECV_IN_ORDER_ALIGNED_128_BYTES:
+		ret = efa_base_ep_support_op_in_order_aligned_128_bytes(&rxr_ep->base_ep, IBV_WR_SEND);
```

Comment:
This is not right. The fact that base_ep support send 128 bytes does not guarantee rdm endpoint support 128 bytes in order send. Because rdm endpoint use device's send, read, localread, and gdrcopy to implement send/receive. To support 128 in_order sendrecv, all of the components must support. Let's simply return FI_EOPNOTSUPP here.